kubernetes/run-e2e.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
+echo -e '- Staring wr Kubernetes End-to-End tests'
+# Environment variables.
+echo -e '- Setting up the test environment..'
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+export KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+
+echo ''
+echo -e 'kubernetes version:\t' $(kubectl version -o json | jq .serverVersion.gitVersion)
```

Comment:
I suppose before getting here you should bail out happily if kubectl, jq and any other reqs are not installed.